python/psautohint/otfFont.py

```diff
@@ -634,15 +710,35 @@ def convertBezToT2(bezString):
     if needHintMasks:
         # If there is not a hintsub before any drawing operators, then
         # add an initial first hint mask to the t2Program.
-        if hintMaskList[1].listPos != 0:
-            hBytes = hintMaskList[0].maskByte(hhints, vhints)
-            t2Program.extend(["hintmask", hBytes])
+        if (mm_hint_info is None) or (not mm_hint_info.defined):
+            # a single font and a reference font for mm hinting are
+            # processed the same way
+            if hintMaskList[1].listPos != 0:
+                hBytes = hintMaskList[0].maskByte(hhints, vhints)
+                t2Program.extend(["hintmask", hBytes])
+                if in_mm_hints:
+                    hm = RefHintMask(pos=0, hbytes=hBytes)
+                    mm_hint_info.hint_masks.append(hm)
+
+            # Convert the rest of the hint masks
+            # to a hintmask op and hintmask bytes.
+            for hintMask in hintMaskList[1:]:
+                pos = hintMask.listPos
+                hBytes = hintMask.maskByte(hhints, vhints)
+                t2List[pos] = [["hintmask"], hBytes]
+                if in_mm_hints:
+                    hm = RefHintMask(pos=pos, hbytes=hBytes)
+                    mm_hint_info.hint_masks.append(hm)
+        elif (mm_hint_info is not None):
+            # This is a MM region font: apply hint masks from reference font.
+            hm0_mask = mm_hint_info.hint_masks[0].hbytes
+            if type(t2List[0][0]) == HintMask:
```

Comment:
Use `isinstance()` instead of `type()`, as suggested by Codacy

Reply:
Done.